autoload/cmake4vim.vim

```diff
@@ -177,4 +177,19 @@ function! cmake4vim#SelectBuildType(buildType) abort
 
     call cmake4vim#GenerateCMake()
 endfunction
+
+function! cmake4vim#RunTarget() abort
```

Comment:
Can we add ability to run this function with arguments? It should allow to pass any arguments to application.
```suggestion
function! cmake4vim#RunTarget(...) abort
```